int32 id
float32 value

# enum: ABS, REL
int16 coord_system

float32 timeout
